test/forms/UploadSupportingMaterialMultipleFormProviderSpec.scala

```diff
@@ -27,27 +26,28 @@ class UploadSupportingMaterialMultipleFormProviderSpec extends StringFieldBehavi
 
   val form = new UploadSupportingMaterialMultipleFormProvider()()
 
-  ".value" must {
-
-    val fieldName = "value"
-
-    behave like fieldThatBindsValidData(
-      form,
-      fieldName,
-      stringsWithMaxLength(maxLength)
-    )
-
-    behave like fieldWithMaxLength(
-      form,
-      fieldName,
-      maxLength = maxLength,
-      lengthError = FormError(fieldName, lengthKey, Seq(maxLength))
-    )
-
-    behave like mandatoryField(
-      form,
-      fieldName,
-      requiredError = FormError(fieldName, requiredKey)
-    )
-  }
+  // TODO - Add Form Tests once we decide which fields need to be within it.
```

Comment:
Do it !

Reply:
Think we can just delete this as there is no fields in the form